### DISCUSSION TITLE
How do I target a portal element?

### DISCUSSION THREAD

**Discussion by showduhtung** [gilbarbara/react-joyride#759] [Q&A]
Hello! Trying to add react-joyride to mui based app, where one of the component is a Mui Drawer. Any idea on a strategy of accessing the portal, as the MUI drawer sits outside the scope of the parent component?

**Comment by magda5281** [comment]
I have the same problem. I start the tour on a page and then one of the buttons opens portal. I'd like to be able to click on, open portal and continue a tour in portal.
